src/odemis/gui/conf/data.py

```diff
@@ -691,6 +701,14 @@
                 "tooltip": u"Current temperature of the spectrometer CCD",
             }),
         )),
+    stream.FastEMSEMStream:
+        OrderedDict((
+            # Display the adjusted pixelsize (pixelsize * scale) with the hw VAs.
+            ("pixelSize", {
+                "label": "Pixel size",
```

Comment:
Could you drop this line? It's not necessary, as it should be the default label. Or, if I'm guessing wrong, please add a comment explaining the label otherwise.